### ISSUE TITLE
ERROR after updating to 2.0.0

### ISSUE THREAD

**Issue by forliuyifei** [YoshinoriN/hexo-tag-config#82]
```
ERROR {
  err: [Error: EISDIR: illegal operation on a directory, read] {
    errno: -21,
    code: 'EISDIR',
    syscall: 'read'
  }
} Plugin load failed: %s hexo-tag-config
```

**Comment by forliuyifei** [comment]
![Screenshot_20201219-124843_Termux](https://user-images.githubusercontent.com/53306807/102681021-d075ad80-41f8-11eb-9557-ae76d5f674e6.png)

**Comment by yoshinorin** [comment]
@forliuyifei 
Thank you for your reporting :)
I will check later.

**Comment by yoshinorin** [comment]
@forliuyifei 
Sorry, I made mistake when published `v2.0.0` 😓 
I have already published `v2.0.1`

Would you please update to `v2.0.1`?

```sh
$ npm install hexo-tag-config@2.0.1
```

Thank you.

**Comment by forliuyifei** [comment]
> @forliuyifei
> Sorry, I made mistake when published `v2.0.0` 😓
> I have already published `v2.0.1`
> 
> Would you please update to `v2.0.1`?
> 
> ```shell
> $ npm install hexo-tag-config@2.0.1
> ```
> 
> Thank you.

Okay, the problem has been solved and it works now
